cluster/addons/calico-policy-controller/calico-node-vertical-autoscaler-deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       priorityClassName: system-cluster-critical
       containers:
-        - image: k8s.gcr.io/cpvpa-amd64:v0.8.1
+        - image: eu.gcr.io/k8s-artifacts-prod/cpa/cpvpa-amd64:v0.8.2
```

Comment:
I don't think we want to specify the `eu` region specifically here. Is there no region agnostic image we can use?

Reply:
/cc @listx 
 I believe we can flip back to use `k8s.gcr.io` once the vanity domain flip [1] happens?

[1] https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/Vanity-Domain-Flip.md

Reply:
That dog suggests that the vanity domain will still be pointed at a specific regional image, though? 

I thought GCR allowed pulling from `gcr.io` (without an explicit region) so that clusters running in GCP do not need to pull images from other regions. This seems to break that?

Reply:
> That doc suggests that the vanity domain will still be pointed at a specific regional image, though?

Correct

> I thought GCR allowed pulling from `gcr.io` (without an explicit region) so that clusters running in GCP do not need to pull images from other regions.

This is not a GCR-wide thing. It is a special privilege granted by the GCR folks to a handful of GCP projects (`google-containers` being one of them, and later, `k8s-artifacts-prod`).